### ISSUE TITLE
Unable to render polymer-2-starter-kit in browser 

### ISSUE THREAD

**Issue by bhargavkonkathi** [Polymer/polymer#4531]
Hello Team,

I followed instructions that are mentioned here [https://www.polymer-project.org/2.0/start/install-2-0](https://www.polymer-project.org/2.0/start/install-2-0). I installed polymer `polymer-cli@next`. My current version of polymer is `0.18.0`, and using following command `polymer init` selected `polymer-2-starter-kit` option. After polymer serve I see following error in browser console. None of the html rendered in the browser.

- Error
`Uncaught TypeError: window.HTMLImports.whenReady is not a function - document-watcher.js:106`

Above error is coming in following options also `polymer-2-application`.

In polymer-2-element option in component html this line of code not replacing with mentioned element name, If I given hello-world generated code as follows in hello-world.html there small problem with this line of code  `window.customElements.define(MyElement.is, MyElement);`

### Description
`Uncaught TypeError: window.HTMLImports.whenReady is not a function - document-watcher.js:106`

#### Steps to Reproduce

1. Create `hello-world` element using polymer-2-element option after polymer init command
2. Check window.customElements.define(MyElement.is, MyElement); line of code in hello-world.html

#### Expected Results
window.customElements.define(HelloWorld.is, HelloWorld);

#### Actual Results
window.customElements.define(MyElement.is, MyElement);

### Versions
-  polymer --version: 0.18.0

**Comment by arthurevans** [comment]
I can't reproduce the first part of the report—I installed polymer-2-starter-kit and it rendered fine. Did you see any errors during the install of polymer-cli or during the polymer init feedback?

If you run `bower info webcomponents/shadycss`, what version does it show for shadycss? (That's where this error is coming from.)

For the second part, I can see that the CLI does generate a working element, but it doesn't try to match the class name to the element name.

**Comment by bhargavkonkathi** [comment]
I see following version after running `bower info webcomponents/shadycss` I see following version for shadycss  `1.0.0-rc.5`. I didn't see any errors during the install of the polymer-cli or during the polymer init.

**Comment by sorvell** [comment]
On what browser and version are you seeing the issue?

**Comment by bhargavkonkathi** [comment]
I have tested in Chrome latest versions i.e 57.0.2987 and 56 also. Polymer-2-starterkit didn't rendered previously after successful installation of polymer-cli and with bower_components the day I posted. But today I reinstalled polymer-cli and deleted all bower_components and reinstalled all, everything work fine with **polymer-2-starter-kit** . Although when I run `polymer build` command i see only default folder inside build. Does build folder structure and build process changed in polymer 2.x when compared to 1.x ?
Thank you team for quick response.
